modules/installation-configuration-parameters.adoc

```diff
@@ -1023,6 +1023,10 @@ You can also use this property to enable the QEMU guest agent by including the `
 |IP addresses for external DNS servers that cluster instances use for DNS resolution.
 |A list of IP addresses as strings. For example, `["8.8.8.8", "192.168.1.12"]`.
 
+|`platform.openstack.loadbalancer`
+|Whether or not to use the default, internal load balancer. If the value is set to `UserManaged`, this default load balancer is disabled so that you can deploy a cluster that uses an external, user-managed load balancer. If the parameter is set, or if the value is `OpenShiftManagedDefault`, the cluster uses the default load balancer.
```

Comment:
> If the parameter is set

Should this be _If the parameter is **_not_** set_?

Reply:
right, is not set

Reply:
```suggestion
|Whether or not to use the default, internal load balancer. If the value is set to `UserManaged`, this default load balancer is disabled so that you can deploy a cluster that uses an external, user-managed load balancer. If the parameter is not set, or if the value is `OpenShiftManagedDefault`, the cluster uses the default load balancer.
```